plugins/modules/zos_copy.py

```diff
@@ -2273,6 +2346,37 @@ def run_module(module, arg_def):
         # Copy to sequential data set (PS / SEQ)
         # ---------------------------------------------------------------------
         elif dest_ds_type in data_set.DataSet.MVS_SEQ:
+            if src_ds_type == "USS" and not is_binary:
+                # Before copying into the destination dataset, we'll make sure that
+                # the source file doesn't contain any carriage returns that would
+                # result in empty records in the destination.
+                # Due to the differences between encodings, we'll normalize to IBM-037
```

Comment:
Good to normalize and this is just a FYI for others who might want to know, if i recall, `IBM -037`, `IBM-1047` and `ISO-8859` all have the same characters. Where they differ is that the left square bracket `[` and right square bracket `]` are going to be different hexadecimal (hex) values.